app/views/provider/admin/authentication_providers/show.html.slim

```diff
@@ -1,5 +1,5 @@
 - content_for :page_header_title do
-  i.fa> class=("fa-#{@authentication_provider.kind}")
+  i.fa> class=("fa-#{@authentication_provider.kind} fa-fw")
```

Comment:
Shouldn't we remove the icons here also?

Reply:
Correct!